modules/auxiliary/admin/misc/clamav_control.rb

```diff
@@ -0,0 +1,55 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => 'ClamAV Remote Command Transmitter',
+        'Description'    => %q(
+          In certain configurations, ClamAV will bind to all addresses and listen for commands.
+          This module sends properly-formatted commands to the ClamAV daemon if it is in such a
+          configuration.
+        ),
+        'Author'         => [
+          'Alejandro Hdeza', # DISCOVER
+          'bwatters-r7',     # MODULE
+          'wvu'              # GUIDANCE
+        ],
+        'License'        => MSF_LICENSE,
+        'References'     => [
+          [ 'URL', 'https://twitter.com/nitr0usmx/status/740673507684679680/photo/1' ]
```

Comment:
Could you add a documentation reference url - https://github.com/vrtadmin/clamav-faq/raw/master/manual/clamdoc.pdf. The **clamd** part of the [documentation]( https://github.com/vrtadmin/clamav-faq/raw/master/manual/clamdoc.pdf) shows us core commands. ex: **`QUIT`** can also exit clamd daemon.